### ISSUE TITLE
macos-notifications fails to install on CPython 3.11

### ISSUE THREAD

**Issue by dstromberg** [Jorricks/macos-notifications#4]
macos-notifications itself may be OK.  But it's probably insisting on having a version of pyobjc-core that is too old for 3.11.  Maybe a setup.py problem?

pyobjc-core installs fine on 3.11 if you don't constrain what version it gets - that is, if you install it by itself.  But installing macos-notifications tries to install pyobjc-core, and that fails.

**Comment by Jorricks** [comment]
Hello @dstromberg,

Thanks for opening this issue!
What would you propose to change it to in terms of versioning?

**Comment by gsmethells** [comment]
`pyobjc-core>=9.1.1` would be best, as that installs fine for Python 3.11

```
$ pip3 install pyobjc-core
Collecting pyobjc-core
  Downloading pyobjc_core-9.1.1-cp311-cp311-macosx_10_9_universal2.whl (730 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 730.2/730.2 kB 3.7 MB/s eta 0:00:00
Installing collected packages: pyobjc-core
Successfully installed pyobjc-core-9.1.1
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv

[notice] A new release of pip is available: 23.0.1 -> 23.1.2
[notice] To update, run: /usr/local/opt/python@3.11/bin/python3.11 -m pip install --upgrade pip
```

**Comment by Jorricks** [comment]
Should be fixed with 0.1.6
